### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by devndevs**
Combining branches to make one source of truth before we branch off and do individual page styling.

### CODE REVIEW

File: app/javascript/components/components/testimonial.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        onExited={() => setAnimating(false)}
+        key={item.src}
+      >
+        <img src={item.src} alt={item.altText} />
+        <CarouselCaption captionText={item.caption} captionHeader={item.caption} />
+      </CarouselItem>
+    );
+  });
+
+  return (
```
Comment (minachuong) [inline app/javascript/components/components/testimonial.js:63]: It looks like there are two returns in this component

File: app/javascript/components/pages/LoggedOutHome.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    text-align: center;
+  }
+  .cardbody{
+    text-align: center;
+  }
+  hr { display: block; margin-before: 0.5em; margin-after: 0.5em; margin-start: auto; margin-end: auto; overflow: hidden; border-style: inset; border-width: 1px;}
+  
+  
+  
+  .jumbotron {
```
Comment (minachuong) [inline app/javascript/components/pages/LoggedOutHome.scss:20]: Looks like there's some possible unintented over writing of bootstrap/reactstrap styling happening. Best practice is to leave classes from third party libraries alone and then add your own css classes to provide custom styling

**Review by minachuong — APPROVED** [review]
👍 Please see comment

File: app/javascript/components/components/testimonial.js (hunk 1 above)
Comment (minachuong) [reply app/javascript/components/components/testimonial.js:63]: oops nm 😎

Comment (devndevs) [reply app/javascript/components/components/testimonial.js:63]: This is how it came from the react strap site and I did not change the returns

**Pull request merged**